share/examples/bitcoin.conf

```diff
@@ -71,6 +71,12 @@
 # configuration option or the addnode RPC, which have a separate limit of 8 connections.
 #maxconnections=
 
+# Maximum upload bandwidth target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB max throughput per day).
```

Comment:
```suggestion
# Maximum upload bandwidth target in MiB per day (e.g. 'maxuploadtarget=1024' is 1 GiB per day).
```

nit: The "max throughput" is kind of redundant to the part before the parantheses, but I'm fine with either version.

Reply:
I agree, drop "max throughput".

Reply:
You're right, addressed in  947f973